tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "baseUrl": ".",
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "incremental": false,
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "lib": ["es2020", "dom"],
+    "module": "ES6",
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "rootDir": "./",
+    "skipLibCheck": true,
+    "target": "ES6",
```

Comment:
I see you've installed `@types/node` but are not doing anything with it, you could add `"types": ['@types/node']` under `compilerOptions`.